deploy/cr.yaml

```diff
@@ -145,7 +145,39 @@ spec:
   port: "5432"
 # The number of replicas to create after a PostgreSQL primary
 # is first initialized. This only works on create
-  replicas: "0"
+#  replicas: "0"
+  pgReplicas:
+    hotStandby:
+      size: 0
+#      resources:
+#        requests:
+#          cpu: "1"
+#          memory: "128Mi"
+#        limits:
+#          cpu: "1"
+#          memory: "128Mi"
+#      volumeSpec: 
+#        accessmode: ReadWriteOnce
+#        matchLabels: ""
+#        name: ""
+#        size: 1G
+#        storageclass: ""
+#        storagetype: dynamic
+#        supplementalgroups: ""
+#      affinity:
+#        default: null
+#      labels:
+#        pg-cluster-label: cluster1
+#      annotations:
+#        pg-cluster-annot: cluster1
+#      enableSyncStandby: false
```

Comment:
it still does not work :( 

```
+ Cluster: cluster1 (6969899893134573720) ----+---------+---------+----+-----------+-----------------+-------------------+------+
| Member                          | Host      | Role    | State   | TL | Lag in MB | Pending restart | Scheduled restart | Tags |
+---------------------------------+-----------+---------+---------+----+-----------+-----------------+-------------------+------+
| cluster1-6bcbf4cccd-8c5sj       | 10.4.2.6  | Leader  | running |  1 |           |                 |                   |      |
| cluster1-repl1-8b74c5d45-xjvzz  | 10.4.2.9  | Replica | running |  1 |         0 |                 |                   |      |
| cluster1-repl2-568979d786-pk9jx | 10.4.0.15 | Replica | running |  1 |         0 |                 |                   |      |
| cluster1-repl3-894756dbd-dhh6l  | 10.4.1.11 | Replica | running |  1 |         0 |                 |                   |      |
+---------------------------------+-----------+---------+---------+----+-----------+-----------------+-------------------+------+
```

But it works when I use old option 'syncReplication: true'

```
+ Cluster: cluster1 (6969910795124367517) ----+--------------+---------+----+-----------+-----------------+-------------------+------+
| Member                          | Host      | Role         | State   | TL | Lag in MB | Pending restart | Scheduled restart | Tags |
+---------------------------------+-----------+--------------+---------+----+-----------+-----------------+-------------------+------+
| cluster1-6f56f7db9f-sqmwf       | 10.4.2.18 | Leader       | running |  1 |           |                 |                   |      |
| cluster1-repl1-575c9dbb78-4vfvx | 10.4.0.32 | Sync Standby | running |  1 |         0 |                 |                   |      |
| cluster1-repl2-6b59df9565-c5mjf | 10.4.2.19 | Replica      | running |  1 |         0 |                 |                   |      |
| cluster1-repl3-894756dbd-cvks7  | 10.4.1.18 | Replica      | running |  1 |         0 |                 |                   |      |
+---------------------------------+-----------+--------------+---------+----+-----------+-----------------+-------------------+------+
```

Reply:
https://github.com/percona/percona-postgresql-operator/pull/50/commits/35d324887677775fcc5696fb0d5c41290cdd92d0